egov-indexer/open-hearing-indexer.yml

```
@@ -0,0 +1,33 @@
+ServiceMaps:
+  serviceName: Hearing Service
+  version: 1.0.0
+  mappings: 
+    - topic: open-hearing-topic
+      configKey: INDEX
+      indexes: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)